### ISSUE TITLE
Game crashes when trying to start a level (any).

### ISSUE THREAD

**Issue by sirzaxxon** [xemu-project/xemu#1291]
### Title

https://xemu.app/titles/544d0002/#Robotech-Battlecry

### Bug Description

Game crashes when trying to load levels (story and training), even after disabling hard FPU (within the UI).

ERROR:/xemu/tcg/i386/tcg-target.c.inc:2428:tcg_out_op: assertion failed: (0)
Bail out! ERROR:/xemu/tcg/i386/tcg-target.c.inc:2428:tcg_out_op: assertion failed: (0)

### Expected Behavior

Levels should load after pressing start training, story, etc.

### xemu Version

xemu_version: 0.7.71
xemu_branch: master
xemu_commit: 22db3304a454b2d289a9d5ea488fa3c3dec836f4
xemu_date: Tue Oct 11 18:38:37 UTC 2022

### System Information

OS: Windows 10 Home
CPU: i9-11900KB @ 3.30GHz
Graphics Device: Nvidia Geforce RTX 3060 ti
Graphics Driver: Nvidia 526.47

### Additional Context

Checked TOML file - didn't see a line for hard FPU to disable manually.
When restarting Xemu after a crash, I have to load the game again (it does not retain the game as normal).  Menus are accessible, just not gameplay.
Thank you.

**Comment by HadetTheUndying** [comment]
Can you try this with a few older releases? I was able to play through a good chunk of this game a few months ago but I haven't actively been testing things on xemu recently.

**Comment by sirzaxxon** [comment]
Great suggestion - interesting find:
I just tried version 7.67 (August release), and it worked when I turned off hard FPU (in the UI).

I opened the TOML (config) file to see that there was a line at the bottom for the hard FPU setting that wasn't there before with the current version:
 [perf]
 hard_fpu = false

I then tried running it again with the newest Xemu version 7.71 (since they share the same config file), and it worked!

So I assume that the newest release doesn't write to the config file when you toggle hard FPU.

Cheers?
